NDB.Covid19/NDB.Covid19/ViewModels/CountriesConsentViewModel.cs

```diff
@@ -38,7 +44,13 @@ public class CountriesConsentViewModel
         public static string COUNTRIES_CONSENT_DIALOG_MESSAGE => "COUNTRIES_CONSENT_DIALOG_MESSAGE".Translate();
         public static string COUNTRIES_CONSENT_DIALOG_OK => "COUNTRIES_CONSENT_DIALOG_OK".Translate();
         public static string COUNTRIES_CONSENT_DIALOG_CANCEL => "COUNTRIES_CONSENT_DIALOG_CANCEL".Translate();
-        
+        public static string EU_CONSENT_NEXT_WITH_CONSENT_BUTTON_TEXT => "EU_CONSENT_NEXT_WITH_CONSENT_BUTTON_TEXT".Translate();
+        public static string EU_CONSENT_NEXT_WITHOUT_CONSENT_BUTTON_TEXT => "EU_CONSENT_NEXT_WITHOUT_CONSENT_BUTTON_TEXT".Translate();
```

Comment:
EU_CONSENT_NEXT_ONLYNORWAY_CONSENT_BUTTON_TEXT

Reply:
Used EU_CONSENT_NEXT_ONLY_NORWAY_CONSENT_BUTTON_TEXT for consistency